client/landing/stepper/declarative-flow/hosted-site-migration-v2/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { translate } from 'i18n-calypso';
+import { type Flow } from 'calypso/landing/stepper/declarative-flow/internals/types';
+import { stepsWithRequiredLogin } from '../../utils/steps-with-required-login';
+import { STEPS } from '../internals/steps';
+
+export default {
+	name: 'hosted-site-migration-v2',
```

Comment:
I see some flows import and use the flow constant here. Could we use the `HOSTED_SITE_MIGRATION_V2_FLOW` to avoid the duplicate string?